README.md

```diff
@@ -380,6 +381,19 @@ If you aim to support IE11 a polyfill for some JavaScript features we are using
 
 * [Fetch](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API): https://github.com/github/fetch
 
+### What dependencies do I need if I run Active Login on Linux or in a Linux Docker container?
+
+The ActiveLogin.Authentication.BankId.AspNetCore.QRCoder package has a dependency on package `libgdiplus` on Linux.
```

Comment:
Make the reference to libgdplus a link to https://github.com/mono/libgdiplus.

Reply:
Good idea. Done.